demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -2657,29 +2658,16 @@ def add_ignored_errors_to_list(self, config, section, key, ignored_errors_list):
     def get_error_ignore_list(self, pack_name):
         ignored_errors_list: dict = {}
         if pack_name:
-            pack_ignore_path = get_pack_ignore_file_path(pack_name)
-
-            if os.path.isfile(pack_ignore_path):
-                try:
-                    config = ConfigParser(allow_no_value=True)
-                    config.read(pack_ignore_path)
-
-                    # create file specific ignored errors list
-                    for section in config.sections():
-                        if section.startswith("file:"):
-                            file_name = section[5:]
-                            ignored_errors_list[file_name] = []
-                            for key in config[section]:
-                                self.add_ignored_errors_to_list(
-                                    config, section, key, ignored_errors_list[file_name]
-                                )
-
-                except MissingSectionHeaderError:
-                    pass
-            else:
-                logger.info(
-                    f"[red]Could not find pack-ignore file at path {pack_ignore_path}[/red]"
-                )
+            if config := get_pack_ignore_content(pack_name):
+                # create file specific ignored errors list
+                for section in config.sections():
+                    if section.startswith("file:"):
+                        file_name = section[5:]
+                        ignored_errors_list[file_name] = []
+                        for key in config[section]:
+                            self.add_ignored_errors_to_list(
+                                config, section, key, ignored_errors_list[file_name]
+                            )
```

Comment:
don't we have a function that reads the config file this way somewhere, that we can use? 🤔

Reply:
as far as i know we don't have, but ill double check.